internal/app/go-filecoin/node/node.go

```diff
@@ -568,7 +596,7 @@ func (node *Node) setupProtocols() error {
 	// DRAGONS: uncomment when we have a storage market
 	//smc := storage.NewClient()
 	//smcAPI := storage.NewAPI(smc)
-	//node.StorageProtocol.StorageAPI = &smcAPI
+	//node.StorageProtocol.StorageClientAPI = &smcAPI
```

Comment:
I am confused with the state we are leaving this in.  It looks like the commented out code contradicts the submodule initialization code above.  Ideally this inconsistency gets resolved in this PR by deciding on an approach and removing hints towards other approaches (i.e. this comment if it makes sense to).

Reply:
good call. Removed this code. May revisit later.